tests/writer_test.py

```diff
@@ -1227,6 +1237,69 @@ def test_write_background_image(self):
             ),
         )
 
+    def test_write_official_reference_v2(self):
+        name = "GlyphsFileFormatv2.glyphs"
+
+        path = os.path.join(DATA, name)
+        font = classes.GSFont(path)
+        with open(path, encoding="utf=8") as file:
+            expected = file.read()
+
+        path = os.path.join(self.temporary_path, name)
+        font.save(path)
+        with open(path, encoding="utf=8") as file:
+            actual = file.read()
+
+        exceptions = [
```

Comment:
I suppose we can't do anything about these 'exceptions'?

Reply:
Well, we can. It is not so much about ordering. So one could tackle it separately to not mix topics. My objective was to capture the behavior.

Reply:
I added several missing keys to the GlyphsFileFormatv3.glyphs. I also added some (hacked together) skripts that help getting all keys from a bunch of files.